pkg/join/join.go

```diff
@@ -212,3 +223,35 @@ func isValidCustomCoreDNSConfig(corednsCustomConfigMap string) error {
 
 	return nil
 }
+
+func ensureUniqueCluster(ctx context.Context, clusterID string, brokerProducer client.Producer, brokerNamespace string,
+	localProducer client.Producer, operatorNamespace string, status reporter.Interface,
+) error {
+	err := localProducer.ForGeneral().Get(ctx, controllerClient.ObjectKey{
+		Namespace: operatorNamespace,
+		Name:      clusterID,
+	}, &submarinerv1.Cluster{})
+
+	if !resource.IsNotFoundErr(err) {
```

Comment:
if found here? should we go to next step?

Reply:
No that means submariner was already installed. We only want to check for dup cluster ID if it's a new install.